meta-ti-foundational/recipes-demos/dsp-offload/ti-librpmsg-dma-example_1.0.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "Audio Offload Example using RPMsg DMA library"
+SUMMARY = "RPMsg DMA shared library"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=744e63d2bb8c6151dcdd97f49aa02c53"
+
+SRC_URI = "git://github.com/TexasInstruments/rpmsg-dma.git;protocol=https;branch=main"
+SRCREV = "609fd72f458ba56f80d06810509ef88d010d2c03"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+DEPENDS = "ti-librpmsg-dma fftw libsndfile1 alsa-lib"
+
+# Install to /lib and /bin instead of default /usr
```

Comment:
Why? The correct spots are `/usr/bin` and `/usr/lib`, so CMake does the right thing by default, not need to change that.